bedrock/thunderbird/templates/thunderbird/includes/top-menu.html

```diff
@@ -0,0 +1,25 @@
+<nav id="nav-main" role="navigation">
+  <span class="toggle" role="button" aria-controls="nav-main-menu" aria-expanded="false" tabindex="0">{{_('Menu')}}</span>
+  <ul id="nav-main-menu" class="has-submenus">
+    <li class="first" id="nav-main-features"><a href="{{ url('thunderbird.index') }}" tabindex="0" aria-owns="nav-main-features-submenu" aria-haspopup="true">{{_('Thunderbird')}}</a>
+      <ul aria-expanded="false" id="nav-main-features-submenu" class="submenu">
+        <li class="first"><a href="{{ url('thunderbird.channel') }}" tabindex="-1">{{_('Releases')}}</a></li>
+        <li><a href="{{ url('thunderbird.features') }}" tabindex="-1">{{_('Features')}}</a></li>
+        <li class="last"><a href="{{ url('thunderbird.organizations') }}" tabindex="-1">{{_('Thunderbird for Organizations')}}</a></li>
+      </ul>
+    </li>
+    <li id="nav-main-addons">
+      <a href="https://addons.mozilla.org/thunderbird/" tabindex="0" aria-owns="nav-main-addons-submenu" aria-haspopup="true">{{_('Add-ons')}}</a>
+      <ul aria-expanded="false" id="nav-main-addons-submenu" class="submenu">
+        <li class="first"><a href="https://addons.mozilla.org/thunderbird/extensions/" tabindex="-1">{{_('Add-ons')}}</a></li>
+        <li class="last"><a href="https://addons.mozilla.org/thunderbird/themes/" tabindex="-1">{{_('Themes')}}</a></li>
+      </ul>
+    </li>
+    <li class="last" id="nav-main-support">
+      <a href="https://support.mozilla.org/products/thunderbird" tabindex="0" aria-owns="nav-main-addons-submenu" aria-haspopup="true">{{_('Support')}}</a>
+      <ul aria-expanded="false" id="nav-main-addons-submenu" class="submenu">
```

Comment:
Is there a reason to have this drop down on support? As the main text and the drop down links to the same page, perhaps we just need the main menu entry?